streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -71,25 +79,28 @@
 public class GlobalStateManagerImplTest {
 
 
-    private final MockTime time = new MockTime();
-    private final TheStateRestoreCallback stateRestoreCallback = new TheStateRestoreCallback();
-    private final MockStateRestoreListener stateRestoreListener = new MockStateRestoreListener();
-    private final String storeName1 = "t1-store";
-    private final String storeName2 = "t2-store";
-    private final String storeName3 = "t3-store";
-    private final String storeName4 = "t4-store";
-    private final TopicPartition t1 = new TopicPartition("t1", 1);
-    private final TopicPartition t2 = new TopicPartition("t2", 1);
-    private final TopicPartition t3 = new TopicPartition("t3", 1);
-    private final TopicPartition t4 = new TopicPartition("t4", 1);
-    private GlobalStateManagerImpl stateManager;
-    private StateDirectory stateDirectory;
-    private StreamsConfig streamsConfig;
-    private NoOpReadOnlyStore<Object, Object> store1, store2, store3, store4;
-    private MockConsumer<byte[], byte[]> consumer;
-    private File checkpointFile;
-    private ProcessorTopology topology;
-    private InternalMockProcessorContext processorContext;
+    final MockTime time = new MockTime();
```

Comment:
instead of `package private` could these variables get `proctected` access?

Reply:
@bbejeck fixed and pushed, thanks for the feedback